### PR TITLE
M3-3641 Accessibility: External links and new windows

### PR DESCRIPTION
**Pull request by alioso**
M3-3641 Accessibility: Present users with friendly message for new window or external site

This PR helps users with screen readers know that they may be clicking on an external link and directed somewhere else, or that the focus is going to switch to a new window (ex: Lish)

## Type of Change
- Non breaking change ('update', 'change')

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodesDetail/MutateDrawer/MutateDrawer.tsx [hunk 1]
```diff
@@ -177,7 +177,12 @@ class MutateDrawer extends React.Component<CombinedProps, State> {
          */}
         <Typography style={{ display: 'none' }}>
           {`Need help? Refer to the `}
-          <a href="google.com" target="_blank" rel="noopener noreferrer">
+          <a
+            href="google.com"
```
Comment (WilkinsKa1) [inline packages/manager/src/features/linodes/LinodesDetail/MutateDrawer/MutateDrawer.tsx]: Unrelated, this seems like it should stay hidden. I'm actually unclear on where this drawer can even be initiated, but just to be safe.

**Comment by WilkinsKa1** [comment]
This is likely an issue with me + my screenreader (using ChromeVox extension), but the `ariadescribedby` attribute is inconsistently described to me (typically the initial focus won't describe, if I focus away and refocus again it'll describe the label, element, etc. and then provide the description as I would expect). Again, I think this is more me not fully utilizing the capabilities of the screen reader but figured I'd mention it.

File: packages/manager/src/features/linodes/LinodesDetail/MutateDrawer/MutateDrawer.tsx (hunk 1 above)
Comment (alioso) [reply packages/manager/src/features/linodes/LinodesDetail/MutateDrawer/MutateDrawer.tsx]: @WilkinsKa1 not being used anywhere so far, so no worries about this

**Comment by alioso** [comment]
@WilkinsKa1 i confirm that vox has some issues and is not fully reliable. i think it is ok to do some exploration, but windows 10 + NVDA seems like the safest path to test accurately (confirming the links contain the external notice

**Pull request merged**